### DISCUSSION TITLE
Statusseite : grüne Leuchte blinkt nicht bei Überschussenergie

### DISCUSSION THREAD

**Discussion by mh79hf** [camueller/SmartApplianceEnabler#600] [Allgemein]
Hallo camueller,
vielen Dank für das tolle Tool. Installation und Konfiguration klappte soweit sehr gut. Nun habe ich einen
schaltbaren Verbraucher konfiguriert. Dieser funktioniert soweit auch. Mich verwundert nur, dass die grüne Lampe 
im Betrieb nicht blinkt obwohl sehr sicher Überschussenergie verwendet wird.  

Woran könnte das liegen?

Viele Grüße MH79HF

**Comment by camueller** [comment]
Das Blinken bei Nutzung von Überschussenegerie gilt nur für Verbraucher mit variabler Leistungsaufnahme (Wallbox, Geräte mit Stufenschalter oder PWM-Schalter).

**Reply by mh79hf** [reply]
Hallo camueller,
danke für die Information. Ich verwende in dem Fall einen HTTP Schalter. Zwischendurch fängt der Schalter auch das Blinken an.
Ich konnte aber keinen Auslöser dazu erkennen. PV Überschuss war lange Zeit da. Akku voll usw ... Mit erreichen der Abschaltzeit war es dann vorbei. 
Vg MH79HF

**Comment by mh79hf** [comment]
Hier nochmal der blinkende HTTP Schalter. Noch jemand eine Idee warum er sporadisch blinkt?
Kann ich die Ursache in den logs finden? Wonach müsste ich suchen?
Vg MH79HF

https://github.com/camueller/SmartApplianceEnabler/assets/167847204/4a70de5a-dcf6-4061-b52d-73fbb5a5da85

**Reply by camueller** [reply]
Eigentlich ergibt es das doch auch aus der Anzeige: min. Laufzeit ist 0 - also bedeutet jede Einschaltung, dass dafrü Überschussenergie verwendet wird.

**Reply by mh79hf** [reply]
Das habe ich soweit verstanden. Bin nur verwirrt, weil das Verhalten (Blinken) nur sporadisch auftritt und nach deinem Post von vor 5 Tagen gar nicht für die Schalterart (HTTP ohne variable Leistungsaufnahme)  vorgesehen ist. Daher nochmal die Rückfrage :)
